site/hooks/useStreams.js

```diff
@@ -52,12 +53,13 @@ export const useStreams = function () {
 
   const { data, error, mutate } = useSWR(
     active ? [`${account}-streams`, library] : null,
-    (_, library) => getStreams(library, account).catch(console.error),
+    // eslint-disable-next-line no-unused-vars
+    ([_, library]) => getStreams(library, account).catch(console.error),
```

Comment:
Same as [this comment](https://github.com/hemilabs/pure.finance/pull/1/files#r1526700826)

Reply:
here: https://github.com/hemilabs/pure.finance/pull/1/commits/df53fc81934898b30ca242efe2b83df564cf9ef7